docs/build/provision/_index.md

```diff
@@ -0,0 +1,251 @@
+If you did not initially provide a full `viam-server` configuration in either of these methods, you will be prompted to paste one in when you connect to the WiFi hotspot.
+You can copy a configured machine's configuration by navigating to the [Viam app](https://app.viam.com/), selecting the **Setup** tab for your machine, and clicking the **Copy viam-server config** button.
+
+## Use a pre-install script
+
+When you install the Viam Agent, either manually using the commands above or automatically as part of your fleet's build and deploy process, you can choose to run a pre-install script to perform provisioning steps before deployment to the target machine.
+For example, you could use the pre-install script to configure and deploy to an SD card or other image file, which you can then use as part of your fleet deployment process.
+You can also use this method to generate a local tarball containing the configured deployment, which you could then deploy later, or through a different medium (such as automation, or as the basis for further customer-specific customization).
+
+See [Pre-install Details](https://github.com/viamrobotics/agent-provisioning#short-version-for-raspberry-pi) for detailed instructions.
+
+## Use a provisioning configuration file
+
+When you install the Viam Agent, either manually using the commands above or automatically as part of your fleet's build and deploy process, you can provide a provisioning configuration file to pre-configure how your machine behaves when it first comes online.
+This functionality is provided by the [Agent Provisioning](https://github.com/viamrobotics/agent-provisioning) subsystem, which is included with the Viam Agent installation by default.
+
+To provision your machine, create a <file>/etc/viam-provisioning.json</file> configuration file, resembling the following:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "manufacturer": "Skywalker",
+  "model": "C-3PO",
+  "fragment_id": "2567c87d-7aef-41bc-b82c-d363f9874663",
+  "disable_dns_redirect": true,
+  "hotspot_prefix": "skywalker-setup",
+  "hotspot_password": "skywalker123"
+}
+```
+
+This file configures some basic metadata, specifies a [fragment](/fleet/configure-a-fleet/) to use to configure the machine, and provides the WiFi signal name and password to allow your machine to connect automatically on startup.
+
+For more information, see the [Agent Provisioning Readme](https://github.com/viamrobotics/agent-provisioning#preinstall-details).
+
+## Use the Viam mobile app
+
+You can also use the [Viam mobile application](/fleet/#the-viam-mobile-app), available for download from the [Apple](https://apps.apple.com/us/app/viam-robotics/id6451424162) and [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US) app stores, to connect to the Viam Agent on deployed machines.
+
+See [Mobile App Provisioning](https://github.com/viamrobotics/agent-provisioning#mobile-app-provisioning) for more information.
```

Comment:
This is also still private... @nataliajacobowitz can we make this public? Can discuss via slack if needed

Reply:
Is it worth just ingesting the info there into here since there isn't much information here and then people don't need to click on a link

Reply:
I would just add this instead: 
Within the Viam mobile app once logged in, navigate to an organization, then to a location, then inside any location tap "Add new smart machine", and follow the instructions in app.